common/test-resources/unit/data/NunitTestResultWithByteOrderMark.xml

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
I think you may have removed the BOM there, was that intentional?

**before:**

```
$ file common/test-resources/unit/data/NunitTestResultWithByteOrderMark.xml
common/test-resources/unit/data/NunitTestResultWithByteOrderMark.xml: XML 1.0 document, UTF-8 Unicode (with BOM) text
```

**after:**

```
$ file common/test-resources/unit/data/NunitTestResultWithByteOrderMark.xml
common/test-resources/unit/data/NunitTestResultWithByteOrderMark.xml: XML 1.0 document, ASCII text
```